src/core/lib/security/credentials/tls/grpc_tls_crl_provider.cc

```diff
@@ -95,6 +95,11 @@ absl::StatusOr<std::shared_ptr<CrlProvider>> StaticCrlProvider::Create(
                        result.status().ToString()));
     }
     std::unique_ptr<Crl> crl = std::move(*result);
+    if (crl_map.contains(crl->Issuer())) {
```

Comment:
This approach requires doing the map lookup twice, once for `contains()` and then again for `crl_map[crl->Issuer()]`.  Instead, I suggest the following:

```
bool inserted = crl_map.emplace(crl->Issuer(), std::move(crl)).second;
if (!inserted) {
  gpr_log(GPR_ERROR, ...);
}
```

Note that this will use the *first* entry in case there is a conflict, not the last.  But that should be fine.

Reply:
I like that, made the change